src/web/src/asset_DELETE.ecpp

```diff
@@ -58,15 +59,117 @@ bool database_ready;
+        } catch (const std::exception& e) {
+            std::string err = TRANSLATE_ME ("ID list is not valid");
+            log_error_audit ("Request DELETE asset ids %s FAILED", idsStr.c_str());
+            http_die ("request-param-bad", "ids", idsStr.c_str(), err.c_str());
+        }
+
+        for(const auto& id: ids) {
+            if (!persist::is_ok_name(id.c_str())) {
+                std::string err = TRANSLATE_ME("valid asset name");
+                log_error_audit("Request DELETE asset id %s FAILED", id.c_str());
+                http_die ("request-param-bad", "id", id.c_str(), err.c_str());
+            }
+        }
+        // end sanity checks
+
+        std::unique_lock<std::mutex> lock1(g_deleteAssetMutex);
+
+        std::vector<uint32_t> dbIds;
+        for(const auto& id: ids) {
+            // get id by name
+            int64_t dbid = DBAssets::name_to_asset_id(id);
+            if (dbid == -1) {
+                lock1.unlock();
+                log_error_audit("Request DELETE asset id %s FAILED", id.c_str());
+                http_die("element-not-found", id.c_str());
+            }
+            if (dbid == -2) {
+                lock1.unlock();
+                std::string err =  TRANSLATE_ME("Database failure");
+                log_error_audit("Request DELETE asset id %s FAILED", id.c_str());
+                http_die("internal-error", err.c_str());
+            }
+            dbIds.push_back(dbid);
+        }
+
+        // delete it
+        std::vector<db_a_elmnt_t> rows;
+        auto ret = asset_mgr.delete_item(dbIds, rows);
+        lock1.unlock();
+
+        bool someAreOk = false;
+        for(const auto& pair: ret) {
+            if (pair.second.status != 0) {
+                someAreOk = true;
+            } else {
+                log_error_audit("Request DELETE asset id %s FAILED with error: %s", idsStr.c_str(), pair.second.msg.c_str());
+            }
+        }
+
+        std::string agent_name = utils::generate_mlm_client_id ("web.asset_delete");
+        try {
+            cxxtools::SerializationInfo si;
+            si.setCategory(cxxtools::SerializationInfo::Category::Array);
+
+            std::vector<std::map<std::string, std::string>> response;
+            for(const auto& row: rows) {
+                auto it = std::find_if(ret.begin(), ret.end(), [&](const std::pair<uint32_t, db_reply_t>& pair) {
+                    return pair.first == row.id;
+                });
+
+                if (it != ret.end() && it->second.status != 0) {
+                    send_configure (row, persist::asset_operation::DELETE, agent_name);
+                }
+
+                cxxtools::SerializationInfo val;
+                val.setCategory(cxxtools::SerializationInfo::Category::Object);
+                val.addValue("asset", row.id);
+                if (it != ret.end()) {
+                    if (it->second.status != 0) {
+                        val.addValue("status", "OK");
```

Comment:
I would make the log_info_audit ("Request DELETE asset id %s SUCCESS", idsStr.c_str ()); here for each failled and success